### PR TITLE
Fix bug when get IP mac maps in windows guest

### PR DESCRIPTION
**Pull request by xutian**
Original code has bug that IndexError  will raise when get IP mac map in windows guest,  this patch will fix the bug.

Thanks,
Xu

### CODE REVIEW

**Comment by xutian** [comment]
@CongLi and @pingl2016  would you like to review this same fix please?

**Comment by xutian** [comment]
@pingl2016  thanks your comments, now this function can works on both rhel6 and rhel7 guest. please review.

Thanks,
Xu

File: virttest/utils_net.py [hunk 1]
```diff
@@ -3092,13 +3092,17 @@ def get_linux_ipaddr(session, nic):
...
-        inet = re.findall("inet (\S+)", out, re.M)[0]
-    except IndexError:
+        inet = re.search(inet_regex, out, re.M).groups()[1]
+    except Exception:
         inet = None
-    inet6 = re.findall("inet6 (\S+)", out, re.M)
-    if not inet6:
-        inet6 = [None]
+
+    inet6_regex = "inet6 (addr:)?(\S+)"
```
Comment (pingl2016) [inline virttest/utils_net.py]: Should add a space behind "addr:". Below is an example of ip info on rhel6 host.
switch    Link encap:Ethernet  HWaddr 00:22:19:30:0D:20  
          inet addr:10.xx.xx.xx  Bcast:10.xx.xx.xx  Mask:255.255.252.0
          inet6 addr: xxxx::xxx:xxxx:xxxx:xxx/64 Scope:Link

**Comment by CongLi** [comment]
LGTM,
acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
Merged, thanks @pingl2016  and @CongLi

**Pull request merged**